Source/UZKArchive.m

```diff
@@ -923,8 +920,9 @@ - (BOOL)extractBufferedDataFromFile:(NSString *)filePath
             UZKLogError("Error closing file (code %d): %{public}@", err, detail);
             [welf assignError:innerError code:err
                        detail:detail];
-            return;
         }
+        
+        return;
```

Comment:
@amosavian Regardless of what we decide regarding the `return` statement directly above, this one is unnecessary since it comes at the end of a block. Let's remove it.

Reply:
I thought we should have a `return` statement at the end of c blocks, to prevent undefined behavior.

Reply:
Interesting, I never heard that before. For now I'll remove it, but it's good to keep in mind.